core/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactorySpec.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the AnswerClassificationResultObjectFactory.
+ */
+ 
+describe('Answer Classification Object Factory', function() {
+  var AnswerClassificationResultObjectFactory;
+  beforeEach(module('oppia'));
+
+  beforeEach(inject(function($injector) {
+    AnswerClassificationResultObjectFactory = $injector.get(
+      'AnswerClassificationResultObjectFactory');
+    EXPLICIT_CLASSIFICATION = $injector.get(
+      'EXPLICIT_CLASSIFICATION');
+  }));
+
+  it('should create a new result', function() {
+    var answerClassificationResult = (
+      AnswerClassificationResultObjectFactory.createNew(
+        'defaultOutcome', 1, 0, EXPLICIT_CLASSIFICATION));
```

Comment:
Here the first argument shouldn't be of string type, right?
As Sean said, console.log should be of help. Also look at `AnswerClassificationService.js` and `AnswerClassificationServiceSpec.js` and see how `createNew` is being used.

Reply:
@shubha1593 PTAL